### DISCUSSION TITLE
我收到邮件要求重新进行学术认证，但是当我进入界面时告知不需要验证

### DISCUSSION THREAD

**Discussion by liuxiaoliu6661** [community/community#53133] [GitHub Education]
如标题，我今日收到多封邮件告知需要对我的学术资格进行验证，但当我进入GitHubbenefits时，却又被告知我不需验证

**Comment by samus-aran** [comment]
Hi @liuxiaoliu6661, thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
